packages/mdc-linear-progress/mdc-linear-progress.scss

```diff
@@ -0,0 +1,132 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@import "@material/theme/mixins";
+@import "./keyframes";
+
+.mdc-linear-progress {
+  position: relative;
+  width: 100%;
+  height: 4px;
+  transform: translateZ(0);
+  transition: opacity 250ms linear;
+  overflow: hidden;
+
+  &__bar {
+    animation: none;
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    transform-origin: top left;
+    transition: transform 250ms ease;
+  }
+
+  &__bar-inner {
+    @include mdc-theme-prop(background-color, primary);
+
+    animation: none;
+    display: inline-block;
+    position: absolute;
+    width: 100%;
+    height: 100%;
+  }
+
+  &--accent .mdc-linear-progress__bar-inner {
+    @include mdc-theme-prop(background-color, accent);
+  }
+
+  &__buffering-dots {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    background-image: url(data:image/svg+xml;charset=utf8,%3Csvg%20version%3D%271.1%27%20xmlns%3D%27http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%27%20xmlns%3Axlink%3D%27http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%27%20x%3D%270px%27%20y%3D%270px%27%20enable-background%3D%27new%200%200%205%202%27%20xml%3Aspace%3D%27preserve%27%20viewBox%3D%270%200%205%202%27%20preserveAspectRatio%3D%27none%20slice%27%3E%3Ccircle%20cx%3D%271%27%20cy%3D%271%27%20r%3D%271%27%20fill%3D%27%23e6e6e6%27%2F%3E%3C%2Fsvg%3E);
```

Comment:
NICE! The only thing is apparently you can squeak out a bit more performance by not encoding single quotes or spaces 👍 

https://codepen.io/tigt/post/optimizing-svgs-in-data-uris